src/components/Modal/index.js

```diff
@@ -1,12 +1,13 @@
 import style from './style.css'
+import theme from '../Theme/style.css'
 import ReactModal from 'react-modal'
 import { h, Component } from 'preact'
 import { getCSSMilisecsValue, wrapWithClass } from '../utils'
 
 const MODAL_ANIMATION_DURATION = getCSSMilisecsValue(style.modal_animation_duration)
 
 const WrapperContent = ({children}) =>
-  wrapWithClass(style.content, children)
+  wrapWithClass(style.content, wrapWithClass(theme.step, children))
```

Comment:
Do we really need to wrap each children into two classes? Couldn't we just merge the styles of the two into one?

Reply:
I tried this and it did not work because I could not get the 50% top to play well with the top and bottom margin. I think we can merge them once we move to `min-height: 100%` space everything from the top down.